app/internal/email.py

```diff
@@ -48,10 +49,45 @@ def send(
     background_tasks.add_task(send_internal,
                               subject=subject,
                               recipients=recipients,
-                              body=body)
+                              body=body + content)
     return True
 
 
+def send_email_to_event_participants(
```

Comment:
This function is very messy :(
Try to autoformat using `black` (git precommit hooks) and maybe separate into multiple functions

Reply:
added precommit hooks, I think it looks fine with the documentation.. tell me what you think!!